rick-morty sdk added to lib with types & schemas

ricardomedina96 opened the pull request:
# Task: 

## [Third-parties API's](https://rickandmortyapi.com/)
path: `src/lib/`

zebasAL committed:
rick-morty sdk added to lib with types & schemas

zebasAL committed:
rick-morty sdk added to lib with types & schemas

ricardomedina96 committed:
solve pnpmlock err

ricardomedina96 committed:
solve pnpm err

ricardomedina96 committed:
siu

ricardomedina96 committed:
ADD ENV to turbojson

ricardomedina96 committed:
remove $

ricardomedina96 committed:
replace turbojson

ricardomedina96 committed:
remove generate

ricardomedina96 committed:
migrate turbojson

ricardomedina96 committed:
add Image from next image

ricardomedina96 committed:
cambios

zebasAL commented on apps/web/.env:
```diff
@@ -12,6 +12,10 @@ VERCEL_GIT_COMMIT_SHA=""
 VERCEL_GIT_COMMIT_MESSAGE=""
 VERCEL_GIT_COMMIT_AUTHOR_LOGIN=""
 VERCEL_GIT_COMMIT_AUTHOR_NAME=""
+<<<<<<< HEAD
+=======
+VERCEL_GIT_PULL_REQUEST_ID=""
+>>>>>>> 948f53c (rick-morty sdk added to lib with types & schemas)
```
Resolve **all** `rebase CONFLICTS` @ricardomedina96

zebasAL replied:
-Add .env to git-ignore file

ricardomedina96 committed:
delete env

ricardomedina96 committed:
parent 6138a602cac3d1389d2c4d2ae3d51e29e66689dc
parent e324c81a0bdd6c6f78e572baf2f00fe624e9402b
author Ricardo Medina <rm09961219@gmail.com> 1677563165 -0700
committer Ricardo Medina <rm09961219@gmail.com> 1677563739 -0700

ricardomedina96 committed:
Merge branch 'ricardo' of https://github.com/zebasAL/rick-morty into ricardo

ricardomedina96 committed:
Merge branch 'ricardo' of https://github.com/zebasAL/rick-morty into ricardo

zebasAL reviewed (approved):
Approved
